booster.code-workspace

```diff
@@ -0,0 +1,65 @@
+{
+  "folders": [
+    {
+      "name": "📦 @boostercloud/application-tester",
+      "path": "packages/application-tester"
+    },
+    {
+      "name": "📦 @boostercloud/cli",
+      "path": "packages/cli"
+    },
+    {
+      "name": "📦 @boostercloud/framework-common-helpers",
+      "path": "packages/framework-common-helpers"
+    },
+    {
+      "name": "📦 @boostercloud/framework-core",
+      "path": "packages/framework-core"
+    },
+    {
+      "name": "📦 @boostercloud/framework-integration-tests",
+      "path": "packages/framework-integration-tests"
+    },
+    {
+      "name": "📦 @boostercloud/framework-provider-aws",
+      "path": "packages/framework-provider-aws"
+    },
+    {
+      "name": "📦 @boostercloud/framework-provider-aws-infrastructure",
+      "path": "packages/framework-provider-aws-infrastructure"
+    },
+    {
+      "name": "📦 @boostercloud/framework-provider-azure",
+      "path": "packages/framework-provider-azure"
+    },
+    {
+      "name": "📦 @boostercloud/framework-provider-azure-infrastructure",
+      "path": "packages/framework-provider-azure-infrastructure"
+    },
+    {
+      "name": "📦 @boostercloud/framework-provider-kubernetes",
+      "path": "packages/framework-provider-kubernetes"
+    },
+    {
+      "name": "📦 @boostercloud/framework-provider-kubernetes-infrastructure",
+      "path": "packages/framework-provider-kubernetes-infrastructure"
+    },
+    {
+      "name": "📦 @boostercloud/framework-provider-local",
+      "path": "packages/framework-provider-local"
+    },
+    {
+      "name": "📦 @boostercloud/framework-provider-local-infrastructure",
+      "path": "packages/framework-provider-local-infrastructure"
+    },
+    {
+      "name": "📦 @boostercloud/framework-types",
+      "path": "packages/framework-types"
+    },
+    {
+      "name": "📦 @boostercloud/metadata-booster",
+      "path": "packages/metadata-booster"
+    }
+  ],
+  "settings": {}
+}
```

Comment:
So, in order to add new packages, this is where we set them up?

Reply:
This is for VS Code to open each of the packages as a separate project, but in the same folder. I mean to document this, but in the next PR when I add cool docs stuff